### PR TITLE
Update pkg references to `open-cluster-management.io`

### PR DESCRIPTION
**Pull request by dhaiducek**
- https://github.com/open-cluster-management/backlog/issues/18572

### CODE REVIEW

File: PROJECT [hunk 1]
```diff
@@ -4,5 +4,5 @@ layout:
 plugins:
   manifests.sdk.operatorframework.io/v2: {}
 projectName: governance-policy-status-sync
-repo: github.com/open-cluster-management-io/governance-policy-status-sync
+repo: open-cluster-management.io-io/governance-policy-status-sync
```
Comment (mprahl) [inline PROJECT]:
```suggestion
repo: open-cluster-management.io/governance-policy-status-sync
```

**Comment by mprahl** [comment]
@dhaiducek that new change looks good but could you please take a look at the CI failure?

**Comment by dhaiducek** [comment]
Yes--it looks to be a failure in the placementrule repo. I think this PR should fix it, but that PR is blocked by a FOSSA scan workflow update:
- https://github.com/open-cluster-management-io/multicloud-operators-subscription/pull/60

**Comment by dhaiducek** [comment]
This PR is ready for review.

**Review by mprahl — APPROVED** [review]

**Pull request merged**